src/params/index.ts

```diff
@@ -0,0 +1,23 @@
+export interface TypesParameters {
+  rootTypeName: string
+  filePath: string
+}
+
+export interface Parameters {
```

Comment:
A general naming discussion: **Options** or **Parameters**.

From my totally subjective anecdotal experience, it seems most projects call these things **Options**. E.g. many JS libs will accept an `options` argument that is an object of flags or config.

Reply:
Had the same thoughts. Yeah, honestly feel the same

Reply:
renamed